repository/src/main/java/ru/karalionak/rest/query/OrderedViewQuery.java

```diff
@@ -0,0 +1,83 @@
+package ru.karalionak.rest.query;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrderedViewQuery {
+    private static final QueryHelper queryHelper = new QueryHelper();
```

Comment:
QueryHelper looks very similar to Util class. I would think about refactoring it.